src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -100,11 +103,26 @@ public HeroModel ToHeroModel()
                     {
                         Name = item.Name,
                         Price = item.Price,
-                        Weight = item.Weight
+                        Weight = item.Weight,
                     }).ToList()
-                }
+                },
             };
             return heroModel;
         }
+
+        public void HeroResurrect()
+        {
+            if (_coinsBag >= 20)
+            {
+                _health = _restoreHealth;
+                _coinsBag -= 20;
+                HeroesCreator.LoadFromFile();
+            }
+            else
+            {
+                Display.ErrorText("You don't have 20 Coins to resurrect");
+                Environment.Exit(1);
```

Comment:
Logic related to stopping the game should stay in `Hell`. Here you just need to know if the resurrection was successfully done.